### PR TITLE
release 0.4.1

### PR DESCRIPTION
**Pull request by chfw**
deliver #235, #232. 

#234 could come in later versions of 0.4.x when it will be ready.

### CODE REVIEW

**Comment by chfw** [comment]
"release early and release often"

File: .moban.d/custom_conf.py.jj2 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{% include "conf.py.jj2" %}
+
+master_doc = "index"
```
Comment (ayan-b) [inline .moban.d/custom_conf.py.jj2]: EOL at EOF is missing.

File: docs/conf.py [hunk 2]
```diff
@@ -128,3 +82,5 @@
 ]
 intersphinx_mapping.update({
 })
+
+master_doc = "index"
```
Comment (ayan-b) [inline docs/conf.py]: EOL at EOF is missing.

File: docs/level-18-user-defined-template-types/.moban.cd/data.base.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+nihao: shijie
+hello: shijie
```
Comment (ayan-b) [inline docs/level-18-user-defined-template-types/.moban.cd/data.base.yaml]: EOL at EOF is missing.

File: docs/level-18-user-defined-template-types/data.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+overrides: data.base.yaml
+hello: world
```
Comment (ayan-b) [inline docs/level-18-user-defined-template-types/data.yml]: EOL at EOF is missing.

**Pull request merged**